docs/source/notes/resumption.rst

```diff
@@ -58,7 +58,7 @@ Instead, our trainer supports the ``autoresume=True`` feature. With autoresume,
 
 With autoresume, users can re-submit the _same_ code to the training run, and the trainer will handle finding and resuming from the latest checkpoints. This works well with systems like Kubernetes that automatically resubmit the same job when there is a node failure (due to spot instances as well). For ``autoresume=True`` to work, we require that both a ``save_folder`` and a ``run_name`` be provided. These are used to search for existing checkpoints.
 
-For an example code, see `examples/checkpoint_autoresume.py <https://github.com/mosaicml/composer/blob/dev/examples/checkpoint_autoresume.py>`_.
+For an example code, see `examples/checkpoint_autoresume.ipynb <https://github.com/mosaicml/composer/blob/dev/examples/checkpoint_autoresume.pynb>`_.
```

Comment:
Since this notebook is now on the docs site, can we use a sphinx link? E.g.

```suggestion
For an example code, see the `Checkpoint Autoresumption <examples/checkpoint_autoresume>`_ tutorial.
```